codespell_lib/data/dictionary.txt

```diff
@@ -49087,8 +49216,10 @@ strnad->strand
 strng->string
 strngs->strings
 stroage->storage
-stroe->store
-stroing->storing
+stroe->store, stroke, strobe, strove,
```

Comment:
Also `strop` and `strode` (past of `stride`)?

Reply:
I couldn't find `stroe->strop`. I'll add `strode`.